templates/cluster-template.yaml

```diff
@@ -193,3 +197,17 @@ spec:
             cloud-config: /etc/kubernetes/azure.json
             cloud-provider: azure
           name: '{{ ds.meta_data["local_hostname"] }}'
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: AzureClusterIdentity
+metadata:
+  name: ${CLUSTER_IDENTITY_NAME}
+  namespace: default
+spec:
+  allowedNamespaces: {}
+  clientID: ${AZURE_CLUSTER_IDENTITY_CLIENT_ID}
```

Comment:
would it make sense to keep this as `AZURE_CLIENT_ID` so existing setups (tilt, etc) can reuse the existing var?

Reply:
let me try